sickbeard/postProcessor.py

```diff
@@ -1029,8 +1029,7 @@ def process(self):  # pylint: disable=too-many-return-statements, too-many-local
 
                 # If the file season (ep_obj.season) is bigger than the indexer season (max_season[0][0]), skip the file
                 if int(ep_obj.season) > int(max_season[0][0]):
-                    self._log(u"File has season %s, while the indexer is on season %s. The file may be incorrectly labeled or fake, aborting."
-                              % (str(ep_obj.season), str(max_season[0][0])))
+                    self._log(u"File has season {0!s}, while the indexer is on season {1!s}. The file may be incorrectly labeled or fake, aborting.".format(str(ep_obj.season), str(max_season[0][0])))
```

Comment:
This is part of what I don't like about the automated fix.  All these `!s` format specifiers.  They should only be used in places like here to avoid the forced typecast to `str`
```pyton
"File has season {0!s}, while the indexer is on season {1!s}".format(
   ep_obj.season, max_season[0][0]
)
```